includes/validation/class-amp-validation-manager.php

```diff
@@ -1955,6 +1991,25 @@ public static function print_plugin_notice() {
 	 * @return void
 	 */
 	public static function enqueue_block_validation() {
+		/*
+		 * The AMP_Validation_Manager::post_supports_validation() method is not being used here because
+		 * a post's status for validation checking can change during the life of the editor, such as when
+		 * the user to toggle AMP back on after having turned it off, and then get the validation
```

Comment:
> the user to toggle AMP back on after having turned it off, and then get the validation

the user **toggles** AMP back on after having urned it off, and then **gets** the validation